packages/next-app/src/app/internalmanager/page.tsx

```diff
@@ -156,8 +160,13 @@ function TableRow({
 }: {
   token: {
     __typename?: "UserInternalBalance" | undefined;
-    token: `0x${string}`;
     balance: string;
+    tokenInfo: {
+      __typename?: "Token" | undefined;
+      symbol?: string | null | undefined;
+      address: string;
+      decimals: number;
+    };
```

Comment:
If this is a type coming from the subgraph you should probably import it from the generated types instead of redefining it here